src/commands/register.ts

```diff
@@ -33,23 +33,23 @@ export default class ComponentRegister extends BaseCommand {
     arg: {
       non_sensitive: true,
```

Comment:
Need to pull in rc - I renamed non_sensitive to sensitive and moved into the Flag def. See other cmds after updating.